test/integ_tests/test_ahs_device.py

```diff
@@ -183,6 +183,25 @@ def test_circuit_can_be_called_local_detunings(
         @qml.qnode(dev)
         def circuit():
             ParametrizedEvolution(H, params, t)
-            return qml.sample()
+            return qml.sample(), qml.expval(qml.PauliZ(0))
+
+        circuit()
+
+    def test_qnode_shape(self):
```

Comment:
Would this be better suited in unit tests?

Reply:
I'm not 100% confident, but I believe anything with `qnode` is supposed to go in integration tests (integration with the PennyLane `qnode` pipeline is part of integration)